File: 7-kyu/did-we-win-the-super-bowl-/javascript/solution.js
const didWeWin = plays => {
  let yardageGained = 0;
  for (const play of plays) {
    if (play.length !== 2) {
      break;
    } else if (play[1] === "turnover") {
      return false;
    } else if (play[1] === "sack") {
      yardageGained -= play[0];
    } else if (["run", "pass"].some(p => p === play[1])) {
      yardageGained += play[0];
    }
  }
  return yardageGained > 10;
};

File: 7-kyu/did-we-win-the-super-bowl-/python/solution.py
def did_we_win(plays):
    yards_gained = 0
    for play in plays:
        if not play:
            break
        elif play[1] == 'turnover':
            return False
        elif play[1] == 'sack':
            yards_gained -= play[0]
        elif play[1] in ['run', 'pass']:
            yards_gained += play[0]
    return yards_gained > 10
